lib/ansible/modules/service_facts.py

```diff
@@ -70,18 +70,26 @@
         state:
           description:
           - State of the service.
-          - 'This commonly includes (but is not limited to) the following: C(failed), C(running), C(stopped) or C(unknown).'
+                - 'This commonly includes (but is not limited to) the following: C(running), C(failed), C(dead), C(exited), C(running), C(stopped) or C(unknown).'
           - Depending on the used init system additional states might be returned.
           returned: always
           type: str
           sample: running
         status:
           description:
           - State of the service.
-          - Either C(enabled), C(disabled), C(static), C(indirect) or C(unknown).
+          - Either C(loaded), C(active), C(inactive), C(failed), C(disabled), C(static), C(indirect) or C(unknown).
```

Comment:
the state and status fields are supposed to be generic across init systems, I'm fine with adding optional fields returned when systemd is run, but we should not customize the generic values to systemd specifics.

Reply:
got it, reverting.